.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: ruby
 rvm:
   - 2.0.0
+  - jruby-9.1.5.0
```

Comment:
Unfortunately the warning at startup is still there. But it refers to a file that we're not using so it's really only a tiny annoyance not a problem with the code.